.travis.yml

```diff
@@ -17,7 +17,15 @@ matrix:
           env: COMPOSER_LOWEST="--prefer-lowest"
     fast_finish: true
 
+addons:
+    apt:
+        packages:
+            - language-pack-fr
+            - language-pack-en
+            - language-pack-de
```

Comment:
An error while copying install procedure from Travis doc. I did not remembered which locale were already used.

DE locale removed in last amending commit